### PR TITLE
Remove reference to "Future" in ML.NET docs

### PR DESCRIPTION
**Pull request by aslotte**
## Summary
Removed the notion of future in the ML.NET documentation and added them to the list as they are now available.

Fixes #Issue_Number (if available)
https://github.com/dotnet/machinelearning/issues/4886

Preview Link: https://review.docs.microsoft.com/en-us/dotnet/machine-learning/automate-training-with-cli?branch=pr-en-us-18239

### CODE REVIEW

File: docs/machine-learning/automate-training-with-cli.md [hunk 1]
```diff
@@ -31,7 +31,10 @@ Currently, the ML Tasks supported by the ML.NET CLI are:
 - `binary-classification`
 - `multiclass-classification`
 - `regression`
-- Future: other machine learning tasks such as `recommendation`, `ranking`, `anomaly-detection`, `clustering`
+- `recommendation`
+- `ranking`
+- `anomaly-detection`
+- `clustering`
```
Comment (luisquintanilla) [inline docs/machine-learning/automate-training-with-cli.md:37]:
```suggestion
- `recommendation`
```

**Review by luisquintanilla — CHANGES_REQUESTED** [review]
Thanks for submitting this PR @aslotte. Looks good. 

I added a suggestion with minor changes. Since this doc is about the CLI. From the list, only recommendation is applicable in this case.

@briacht can you confirm.

File: docs/machine-learning/automate-training-with-cli.md (hunk 1 above)
Comment (aslotte) [reply docs/machine-learning/automate-training-with-cli.md:37]: Ah thanks for catching that @luisquintanilla. I missed that this was only for AutoML. I believe you're correct.

**Comment by luisquintanilla** [comment]
@aslotte when you get a chance, if you're comfortable with the changes I've suggested, can you please merge them into your branch so I can approve and merge this PR. Thanks!

**Comment by aslotte** [comment]
@luisquintanilla how does that look?

**Comment by luisquintanilla** [comment]
@aslotte looks good. Sorry to make you take action though. Since the new version of the CLI is out, @briacht has made updates to this doc that reflect the latest experience. I'll close this PR for now and make sure your changes are included as part of her PR #18782 . Thanks again for the contribution!

**Pull request closed without merging**